bin/native_connectors

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+
+require 'native_connectors'
+NativeConnectors.run!
```

Comment:
For some reason I thought that's the executable for native service implementation, but it just lists connectors. What's our plan on these binaries?

Will we have `bin/native_connectors run|list` or something else?

Reply:
This is requested by @tarekziade [here](https://elastic.slack.com/archives/C03GAPTQ2JF/p1663047317660909?thread_ts=1663046607.954099&cid=C03GAPTQ2JF)

where in python, it uses the argument `action`. e.g. 
```
RUN elastic-ingest --action list -c /usr/share/enterprise-search/connectors.yml
```

As I explained in https://github.com/elastic/connectors-ruby/pull/303#issuecomment-1246310410, I'm not a fan of adding `if else` statement to the connector service app, that's why I expose 2 executables.